app/components/UsersActions/ActionDiff.jsx

```diff
@@ -10,17 +10,18 @@ import {
 import formatSeconds from '../../lib/seconds_formatter'
 import EntityTitle from './EntityTitle'
 
-
+@translate('keys')
 class ActionDiff extends PureComponent {
   render() {
     const allActions = this.props.allActions || new List([this.props.action])
     const diff = this.generateDiff(allActions, this.props.action)
+    const t = this.props.t
```

Comment:
Tu ne l'utilises qu'une fois, pas besoin de déconstruire l'objet tu peux faire un `this.props.t(key)` direct